### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by Qix-** [m-ou-se/ninj#29]
Hey there! Just came across this; I use `ninja` heavily but it's not very actively maintained either and was hoping there was a maintained re-implementation of it somewhere.

Either way, this looks like a noble effort. Kudos :)

**Comment by m-ou-se** [comment]
Nope, unfortunately not. I was spending time on this when I was still actively using C++ projects that were compiled with CMake+Ninja. But nowadays I only use Rust, and don't have much use for this myself anymore.

It was a lot of fun to work on though, so might pick it up some day again. :)
